openml-lightgbm-module/lightgbm-lib/pom.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ The copyright of this file belongs to Feedzai. The file cannot be
+  ~ reproduced in whole or in part, stored in a retrieval system,
+  ~ transmitted in any form, or by any means electronic, mechanical,
+  ~ photocopying, or otherwise, without the prior permission of the owner.
+  ~
+  ~ (c) 2020 Feedzai, Strictly Confidential
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.feedzai.openml.lightgbm</groupId>
+    <artifactId>lightgbm-lib</artifactId>
+    <version>2.3.190</version>
```

Comment:
Use the properties module-wide and use here the property lightgbmlib.version.

Reply:
Maven doesn't allow you to use a property to define the modulo version. Either we configure it on jgitver (Not sure if possible) or hardcode it

Reply:
The only reason that on openml-lightgbm-module/openml-lightgbm/pom.xml you have the below code working
    ```
<parent>
        <groupId>com.feedzai.openml.lightgbm</groupId>
        <artifactId>openml-lightgbm-module</artifactId>
        <version>0.0.0-SNAPSHOT</version>
    </parent>
    <groupId>com.feedzai</groupId>
    <artifactId>openml-lightgbm</artifactId>
    <version>${project.version}</version>
```

It's because we use jgitver, that override ${project.version} to the correct version